docs/units/combining_and_defining.rst

```diff
@@ -98,4 +98,22 @@ can be enabled by calling :func:`~astropy.units.add_enabled_units`::
     kmph         | 0.277778 m / s  |         ,
   ]
 
+If new units are defined with prefixes enabled, the prefixed units must be
+explicitly enabled as well, e.g. by using the `namespace` argument::
+
+  >>> new_units = dict()
+  >>> foo = u.def_unit(['Fo', 'foo'], prefixes=True, namespace=new_units)
```

Comment:
Are you sure you don't want a more exciting example, say, https://en.wikipedia.org/wiki/Kessel_Run ?

Reply:
Did you mean to link the article about USAF software development? Or the star warsy one? Either way, I'm not quite sure what you mean. I'm pretty sure parsecs are already in astropy, no? Anyway, I am open to changing the example just not quite sure what you want here :)

Reply:
Haha, I was only joking. I was referring to Star Wars.